### PR TITLE
293 flag material

### PR DESCRIPTION
**Pull request by hafizhanindito**
#293 

### Breakdown:
- [x] Create migration for edit material (add is_used column)
- [x] Add the necessary API for speaker to modify the is_used flag
- [x] Cleaning and linting the commits

### CODE REVIEW

**Comment by hafizhanindito** [comment]
to test the API used token for id=4 (correspond to speaker user)

File: app/controllers/speaker_document_controller.py [hunk 1]
```diff
@@ -62,19 +66,53 @@ def create(request, user):
...
+            if speaker is None:
+                return BaseController.send_error_api(None, 'speaker not found')
+            speaker = speaker.as_dict()
+            speaker_id = speaker['id']            
+            summary = request.form['summary'] if 'summary' in request.form else ''
+            title = request.form['title'] if 'title' in request.form else ''
+            is_used = request.form['is_used'] if 'is_used' in request.form else 0
+            if speaker_id:
+                payloads = {
+                    'speaker_id': speaker_id,
```
Comment (andy-shi88) [inline app/controllers/speaker_document_controller.py]: we should not be able to change the speaker id

Comment (andy-shi88) [reply app/controllers/speaker_document_controller.py]: I'll merge after the changes @hafizhanindito

**Comment by hafizhanindito** [comment]
okay, it should does it

**Pull request merged**